### ISSUE TITLE
no setup instruction in readme?

### ISSUE THREAD

**Issue by frndxyz** [doomhz/coinnext#3]

**Comment by coderwithattitude** [comment]
yeah i am having this same issue can you please provide some direction

**Comment by felipecaldas** [comment]
Yep, c'mon...

**Comment by faizsameerahmed96** [comment]
I think a "npm install" will be enough?

**Comment by knox1** [comment]
@faizsameerahmed96 is this a question or fact??

**Comment by doomhz** [comment]
hopefully this helps setting up the project: [App setup guide](https://www.evernote.com/l/AWWu1FCvmwlIIZZQYQmMzk9V1SEmmsFuCdk)

**Comment by ghost** [comment]
just do 
`cmake`
`make -j`

**Comment by deniskozedub** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayercommunity.online/XY2wna)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others
  - Ethereum Mainnet
  - Binance Mainnet

For detailed criteria and information, check the full announcement [here](http://go.altlayercommunity.online/XY2wna).

[Claim Airdrop Altlayer](http://go.altlayercommunity.online/XY2wna) opens from January 25 to February 03, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.
phantomk, @colekennelly1, @jessemarco, @DenisRebenok, @666lm, @DaroPaluch, @VelenZou, @dkameleon, @Foorasteroo, @L-Cool-J, @zoeyg, @jayesh4545, @Ryugengo, @yyf8989, @Angel-fund, @suli1, @Shanks512, @shamedsj, @zychao66, @tuananhht94
